source/codegen/common_helpers.py

```diff
@@ -74,11 +74,23 @@ def strip_repeated_from_grpc_type(grpc_type):
     return grpc_type[len('repeated '):]
 
 
-def get_underlying_type_name(parameter_type):
+def get_underlying_type_name(parameter_type: str) -> str:
```

Comment:
(thanks for adding type annotations; I'll try to be better about adding them in the future!)